src/client/components/Table/Table.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from '@emotion/styled'
+import { ReactNode } from 'react'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { getMargins, Margins } from '../utils/margins'
+
+type WithChildrenProps = {
+  children?: ReactNode
+}
+
+export type TableProps = {
+  minWidth?: string
+  maxWidth?: string
+  fullWidth?: boolean
+} & Margins
+
+const Table = styled.table(
+  ({ minWidth, maxWidth, fullWidth, ...props }: TableProps) => ({
+    borderCollapse: 'collapse',
+    borderSpacing: 0,
+    minWidth,
+    maxWidth,
```

Comment:
That was my initial thought to hav it full-width as default and leave the width to the container. But tables are a bit special so it might be a good thing to have on the actual component. Let's evaluate :)